src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const nameValue = `name: ${this.name}`;
+      const versionValue = `chip version: ${this.version}`;
+      const wheelsValue = `wheels: ${this.wheels}`;
+
+      return `${nameValue}, ${versionValue}, ${wheelsValue}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(step = 1) {
```

Comment:
formatting is still important, all over your code
```suggestion
    get info() {
      const nameValue = `name: ${this.name}`;
      const versionValue = `chip version: ${this.version}`;
      const wheelsValue = `wheels: ${this.wheels}`;

      return `${nameValue}, ${versionValue}, ${wheelsValue}`;
    },

    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },

    goForward(step = 1) {
```